benchmarks/msg_pingpong.c

```diff
@@ -193,7 +193,7 @@ static int run(void)
 			goto out;
 	}
 
-	ft_finalize();
+	ret = ft_finalize();
 out:
 	fi_shutdown(ep, 0);
```

Comment:
Okay, so should the return status of shutdown also be checked? If so which one gets reported? 

Maybe something like:

```
ret = ft_finalize();
if (ret)
    FT_PRINTERR(...);

return fi_shutdown();
```